autosar/writer/behavior_writer.py

```diff
@@ -114,7 +114,8 @@ def _writeRunnableXML(self,runnable):
                 lines.append(self.indent('<CAN-ENTER-EXCLUSIVE-AREA-REF DEST="%s">%s</CAN-ENTER-EXCLUSIVE-AREA-REF>'%(exclusiveArea.tag(self.version),exclusiveArea.ref),2))
             lines.append(self.indent('</CAN-ENTER-EXCLUSIVE-AREA-REFS>',1))
         if self.version >= 4.0:
-            lines.append(self.indent('<MINIMUM-START-INTERVAL>%d</MINIMUM-START-INTERVAL>'%(int(runnable.minStartInterval)),1))
+            if runnable.minStartInterval is not None:

```

Comment:
Check out the writer for autosar.behavior.TimingEvent how I divided by 1000.0 just before writing the XML

Reply:
Will do